zk/classpath

```diff
@@ -13,4 +13,4 @@
 ../dist/lib/ext/javaee-api.jar
 ../dist/lib/ext/commons-codec.jar
 ../dist/lib/ext/commons-io.jar
-../dist/lib/ext/closure-compiler.jar
+../dist/lib/ext/closure-compiler-unshaded.jar
```

Comment:
closure-compiler-unshaded.jar will need other dependencies, put them in dist/lib/ext and add them here